src/lib/boolean.spec.ts

```diff
@@ -4,6 +4,7 @@ describe("boolean tests", () => {
   test.each([
     // Test cases like in C#
     // https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System.Runtime.Tests/System/BooleanTests.cs
+    // eslint-disable-next-line unicorn/no-null

```

Comment:
probably we can disable the rule completely for the `*.spec.ts` test files

Reply:
I would go a step further and disable it completley. Especially with external libraries it will be a mess.

Reply:
rule is removed and change reverted